### PR TITLE
Feature/health api bridge accounts

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Added health API to check the balance of bridge network account (CBI-1049)

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -26,3 +26,6 @@ BLOCK_PRODUCTION_DIFF=15
 DDC_METRIC_URL=https://node-2.ddc.dev.cere.network/api/rest
 
 ACCESS_KEY_FOR_RESTART=mbHcCRQn9q4xF4D3
+
+BRIDGE_ACCOUNTS=[{"address": "5EYCAe5g7bGpFHagwe26HiRHdHdE3hobrwV6hq1UD2BPAiZb", "walletName": "bridge-account", "thresholdBalance": "10000000000"}, {"address": "5C4x1RWy71WuMNbGSy6GFhFuAWmQD2r2jBdjvaHrhvaqu6eb", "walletName": "relayer-account","thresholdBalance": "100000000000"}]
```
[the remark this answers is not in the document]
Comment (AndreiNavoichyk) [reply .env.local:30]: How are you going to check health for MAINNET and TESTNET? Update the configuration structure to smth like:
{
  {
    network: MAINNET,
    accounts: [{"address": "", "name": "", "threshold": 1}, ...]
  },
  ...
}

File: README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ This application is supposed to be used as API for [Laboratory UI](https://githu
 * Added migrations framework
 * Added migration to clean up Devnet scanned data
 * Add api to find duplicate entries in transaction table
+* Adding health api for bridge relayer accounts
```
Comment (AndreiNavoichyk) [inline README.md:12]: Not for bridge only. Update

File: src/modules/health/health.controller.ts [hunk 3]
```diff
@@ -93,4 +93,19 @@ export class HealthController {
     }
     return this.healthService.nodeDroppedStatus(network);
   }
+
+  @Get('account-threshold/:accountName')
```
Comment (AndreiNavoichyk) [inline src/modules/health/health.controller.ts:97]: Here should be 2 endpoints:
1. To check all accounts balances per network `/balances/NETWORK_NAME`
2. To check some account balance in the network `/balances/NETWORK_NAME/ACCOUNT_NAME`

File: src/modules/health/health.controller.ts [hunk 4]
```diff
@@ -93,4 +93,19 @@ export class HealthController {
...
+  @ApiNotFoundResponse({description: 'Account balance is less than threshold'})
+  public async accountThreshold(@Param('accountName') accountName: string, @Res() res: Response): Promise<any>{
```
Comment (AndreiNavoichyk) [inline src/modules/health/health.controller.ts:99]: Each method should have a verb. Where is it here? Rename it.

**Review by AndreiNavoichyk — CHANGES_REQUESTED** [review]

File: .env.local (hunk 1 above)
[the remark this answers is not in the document]
Comment (karthikeyan-cere) [reply .env.local]: Updated.

File: README.md (hunk 2 above)
Comment (karthikeyan-cere) [reply README.md]: Updated.

File: src/modules/health/health.controller.ts (hunk 3 above)
Comment (karthikeyan-cere) [reply src/modules/health/health.controller.ts]: Updated.

File: src/modules/health/health.controller.ts (hunk 4 above)
Comment (karthikeyan-cere) [reply src/modules/health/health.controller.ts]: Update.d

File: .env.local [hunk 5]
```diff
@@ -26,3 +26,6 @@ BLOCK_PRODUCTION_DIFF=15
 DDC_METRIC_URL=https://node-2.ddc.dev.cere.network/api/rest
 
 ACCESS_KEY_FOR_RESTART=mbHcCRQn9q4xF4D3
+  
+HEALTH_ACCOUNTS_BALANCE=[{"network":"DEVNET", "accounts": [{"address" : "5EYCAe5g7bGpFHagwe26HiRHdHdE3hobrwV6hq1UD2BPAiZb", "name": "bridge-account", "minBalance": 1},{"address" : "5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y", "name": "relayer-account", "minBalance": 10} ]},{"network":"MAINNET", "accounts": [{"address" : "5EYCAe5g7bGpFHagwe26HiRHdHdE3hobrwV6hq1UD2BPAiZb", "name": "bridge-account", "minBalance": 1}]}]
+BRIDGE_ACCOUNTS_NETWORK='DEVNET'
```
Comment (AndreiNavoichyk) [inline .env.local:31]: Clean up

File: src/modules/health/health.service.ts [hunk 6]
```diff
@@ -187,4 +228,43 @@ export class HealthService {
...
+
+  /**
+   * Load Health accounts
+   */
+  private loadHealthAccount() {
+    const healthAccounts = JSON.parse(this.configService.get('HEALTH_ACCOUNTS_BALANCE'));
+    healthAccounts.forEach((element) => {
+      this.accounts.set(element.network, {account: element.accounts});
+    });
+
```
Comment (AndreiNavoichyk) [inline src/modules/health/health.service.ts:240]: Remove empty line.

**Review by AndreiNavoichyk — APPROVED** [review]

File: .env.local (hunk 5 above)
Comment (karthikeyan-cere) [reply .env.local]: done

File: src/modules/health/health.service.ts (hunk 6 above)
Comment (karthikeyan-cere) [reply src/modules/health/health.service.ts]: done

**Pull request merged**